Dockerfile

```diff
@@ -5,17 +5,16 @@ ARG BUILD_DATE
 ARG VCS_REF
```

Comment:
Needs documentation for how to build, deploy, and start the workspace updated. Docs need to cover starting the workspace outside of a docker container.

Reply:
There used to be a docker-compose setup for how to start it using docker, however you asked that it be removed. Are you asking about how to start workspace in a docker container, or about how to start workspace under tomcat?

Reply:
> you asked that it be removed
When did I ask that?

Start with tomcat. A dev needs to have instructions for starting the same config we have in production on local metal. Currently the instructions are for glassfish.

Reply:
I've added a bunch of documentation to docsource/buildandconfigure.rst
We can build on that moving forward.